code/controllers.js

```diff
@@ -65,10 +66,24 @@ const postCommentByArticleId = (req, res, next) => {
     });
 };
 
+const patchArticleByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  addVotes(article_id, inc_votes)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
Just like you've done above - this needs to be sent on an object with a key of article and a value of whatever that article is (this is always the case when sending a response)